src/goblint_lib.ml

```diff
@@ -124,6 +124,7 @@ module RelationPriv = RelationPriv
 module ThreadEscape = ThreadEscape
 module PthreadSignals = PthreadSignals
 module ExtractPthread = ExtractPthread
+module LoopTermination = LoopTermination
```

Comment:
This isn't a concurrency analysis, so it should go to below where `Termination` used to be.

Reply:
I moved it to the analyses / other category, where `Termination` previously was.